aten/src/ATen/native/Convolution.cpp

```diff
@@ -1732,11 +1732,10 @@ std::tuple<Tensor,Tensor,Tensor> _convolution_double_backward( const std::option
   // See [Note: hacky wrapper removal for optional tensor]
   c10::MaybeOwned<Tensor> ggI_maybe_owned = at::borrow_from_optional_tensor(ggI_opt);
   const Tensor& ggI = *ggI_maybe_owned;
-  const Tensor& ggW_r = ggW_r_opt.value_or(Tensor());
+  Tensor ggW = ggW_r_opt.value_or(Tensor());
```

Comment:
the `const Tensor&` here was a lie; std::optional<T>::value_or returns T. In this particular case I'm not even sure temporary lifetime extension applies (https://abseil.io/tips/107); it may (or may not) have created a dangling reference

Reply:
I checked that, const & can reference a temporary object in C++, not an undefined behaviour.